app/grandchallenge/retina_api/urls.py

```diff
@@ -0,0 +1,24 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from grandchallenge.retina_api import views
+from django.views.decorators.cache import cache_page
+from django.conf import settings
+
+app_name = "api"
```

Comment:
I called it just `api` since it is a nested namespace, so the main route of the app is accessed at `http://gc.localhost/retina/api/`. Reversing the urls it is done by `retina:api:<viewname>`

Reply:
Changed it as suggested (bcfa6c91876025c40cfab44051a90809ab79ef89)